src/node.cc

```diff
@@ -2611,6 +2611,30 @@ void SetupProcessObject(Environment* env,
                     "platform",
                     OneByteString(env->isolate(), NODE_PLATFORM));
 
+  // process.release
+  Local<Object> release = Object::New(env->isolate());
+  READONLY_PROPERTY(process, "release", release);
+
+  READONLY_PROPERTY(release, "name", OneByteString(env->isolate(), "iojs"));
+
+#if defined(NODE_RELEASE_SOURCE_URL)
+  READONLY_PROPERTY(release,
+                    "sourceUrl",
+                    OneByteString(env->isolate(), NODE_RELEASE_SOURCE_URL));
+#endif
+
+#if defined(NODE_RELEASE_HEADERS_URL)
+  READONLY_PROPERTY(release,
+                    "headersUrl",
+                    OneByteString(env->isolate(), NODE_RELEASE_HEADERS_URL));
+#endif
+
+#if defined(NODE_RELEASE_LIB_URL)
+  READONLY_PROPERTY(release,
+                    "libUrl",
+                    OneByteString(env->isolate(), NODE_RELEASE_LIB_URL));
+#endif
```

Comment:
Yes ... this is an option but this gives us more flexibility for testing and baking in arbitrary urls to releases including nightlies, I can hardwire the host in here and handle the nightlies special case with macros if you think that's the best way to go. For now this is really helpful for testing this feature so I'll leave it asis.